src/test/java/teammates/test/cases/browsertests/InstructorSubmissionAdjustmentUiTest.java

```diff
@@ -38,7 +38,7 @@ protected void prepareTestData() {
 
         removeAndRestoreDataBundle(testData);
     }
-
+    
```

Comment:
This is also causing the build failure as indicated by the red `x` next to your commits.